curvefs/src/client/rpcclient/task_excutor.h

```diff
@@ -213,6 +233,18 @@ class TaskExecutorDone : public google::protobuf::Closure {
         return excutor_;
     }
 
+    MetaServerClientDone* GetDone() {
+        return done_;
+    }
+
+    void SetInodeAttrs(const RepeatedPtrField<InodeAttr>& inodeAttrs) {
+        dynamic_cast<BatchGetInodeAttrDone *>(done_)->SetInodeAttrs(inodeAttrs);
+    }
+
+    const RepeatedPtrField<InodeAttr>& GetInodeAttrs() const {
```

Comment:
These two functions don't seem to belong to this class, TaskExecutor should not know about BatchGetInodeAttr

Reply:
Yes, miss deleted. fixed.